### ISSUE TITLE
It doesn't work at all

### ISSUE THREAD

**Issue by Dragonsofts** [stereolabs/zed-unreal-plugin#1]
ERROR，ERROR  , all ERROR

**Comment by nesnes** [comment]
Hi,

I'll need a lot more info on the errors to help you!
Notice that the plugin requires a custom version of the unreal engine, all the details are described here: https://docs.stereolabs.com/mixed-reality/unreal/getting-started/

**Comment by pwwarren** [comment]
Can't download custom engine. Getting 404 error at links embedded in guide.

**Comment by tomishninja** [comment]
Can we reopen this issue since the current version of the engine is no longer compliable in windows?

**Comment by obraun-sl** [comment]
What is the error you have when building the engine?
Can you precise the Visual studio version you are using to build it?

**Comment by tomishninja** [comment]
I have used visual studio 2017 and 2019 to do this they both show similar errors when building Unreal Engine 4.2.

I have attached the output log to this code but there are just several minor issues with the code that are not addressed at this point which prevents the system from building several libraries. (MovieSceneTools is one example). My assumption is that the build tools between both versions of this system are shared. 

[log.txt](https://github.com/stereolabs/zed-unreal-plugin/files/4861323/log.txt)
